frontend/src/features/properties/components/forms/subforms/columns.tsx

```diff
@@ -22,15 +22,28 @@ const getEditableMoneyCell = (disabled: boolean | undefined, namespace: string,
   };
 };
 
-const getFiscalYear = (field: string) => {
-  return (cellInfo: any) => {
-    return formatFiscalYear(cellInfo.row.values[field]);
-  };
+/**
+ * Create a formik date picker using the passed cellinfo to get the associated data.
+ * This information is only editable if this cell belongs to a parcel row.
+ * @param cellInfo provided by react table
+ * @param minDate restrict the minimum date that can be selected
+ * @param oldDateWarning warn if the user selects an old date
+ */
```

Comment:
are the minDate and oldDateWarning params supposed to be there? I don't see them